modules/imgcodecs/src/loadsave.cpp

```diff
@@ -176,6 +176,9 @@ struct ImageCodecInitializer
         decoders.push_back( makePtr<Jpeg2KDecoder>() );
         encoders.push_back( makePtr<Jpeg2KEncoder>() );
     #endif
+    #ifdef HAVE_OPENJPEG
+        decoders.push_back( makePtr<Jpeg2KDecoder>() );
```

Comment:
To avoid confusion (for example, in stacktraces) it makes sense to use different name, like `OpenJpegDecoder`.

Reply:
how about `Jpeg2KOpjDecoder`?

Reply:
its Jpeg2KOpjDecoder now